api/webview/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+from django_pgjson.fields import JsonField
+
+
+class Document(models.Model):
+    source = models.CharField(max_length=100)
+    docID = models.CharField(max_length=100)
+
+    providerUpdatedDateTime = models.DateTimeField(null=True)
```

Comment:
Because the way we have the "approved sets" currently working, documents are all processed as raw documents, and then only the ones that are approved are then normalized. We did that in case anyone changed the sets they wanted to approve, so we wouldn't have to scrape any new material. Since the providerUpdatedDateTime is added during normalization, it's never added to docs that are just raw.